chromadb/test/property/test_cross_version_persist.py

```diff
@@ -277,6 +277,8 @@ def test_cycle_versions(
         e = conn1.recv()
         raise e
 
+    p.close()
```

Comment:
context for this addition?

Reply:
Windows wasn't free'ing sqlite connections unless we close() the process